h2o-persist-s3/src/main/java/water/persist/PersistS3.java

```diff
@@ -6,6 +6,7 @@
 import com.amazonaws.Protocol;
 import com.amazonaws.auth.*;
 import com.amazonaws.auth.profile.ProfileCredentialsProvider;
+import com.amazonaws.internal.StaticCredentialsProvider;
```

Comment:
internal? is this safe to use? seems deprecated in more recent version of AWS libs

Reply:
You're right. This one was super easy to replace with our own implementation. So I did it :) 

Just a sidenote: The whole API as we use it is deprecated in newer versions.